testing/test-helper.js

```diff
@@ -25,3 +25,20 @@ export function stubService(sandbox, win, serviceId, method) {
   });
   return stub;
 }
+
+/**
+ * @param {!Document} doc
+ * @param {string} tagName
+ * @param {!Object} attributes
+ * @return {!Element} created element
+ */
+export function createElementWithAttributes(doc, tagName, attributes) {
```

Comment:
I actually found this quite useful, not limited in test. Maybe we can consider moving to `dom.js`